lib/include/openamp/remoteproc_virtio.h

```diff
@@ -39,6 +39,11 @@ extern "C" {
 /* define vdev notification function user should implement */
 typedef int (*rpvdev_notify_func)(void *priv, uint32_t id);
 
+/* Define remoteproc virtio memory management function */
+struct remoteproc_virtio;
+typedef void *(*rpvdev_alloc_buf)(struct remoteproc_virtio *rpvdev, size_t size, size_t align);
```

Comment:
should we replace `struct remoteproc_virtio *rpvdev` with `void *priv` like other callback?

Reply:
Yes, I will update.